### DISCUSSION TITLE
My old account got suspended.

### DISCUSSION THREAD

**Discussion by harshm2601** [community/community#119686] [Other Feature Feedback, Questions, & Ideas]
my account "harshism1" got suspended and i didn't received any mail regarding this. But after that i realized, i might have been hacked as  someone tried hacking my linkedIn account , and as i didn't had my 2FA turned on , hacker might have got into my account and may have violated github policy. I am not able to access github account which is why i created new one. I already created a ticket. But i haven't  got any response yet. can anyone help me??

**Comment by ghostinhershell** [comment]
Hi @harshm2601,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
